### PR TITLE
made improvements in school event

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: school_event/models/event.py [hunk 1]
```diff
@@ -149,19 +149,16 @@ def event_open(self):
     def event_close(self):
         '''Method to change state to close'''
         self.write({'state': 'close'})
-        return True
```
Comment (JayVora-SerpentCS) [inline school_event/models/event.py]: self.state karjo

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Improve the README.rst file of a module. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_schoolevent/school_event/README.rst

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please add active_id inside the browse method. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_schoolevent/school_event/wizard/terminate_reason.py#L13

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please remove line no 6 and add it on line no 5 https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_schoolevent/school_event/models/event.py#L6
Exa.:- from odoo.exceptions import UserError, ValidationError

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please add domain inside the search view. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_schoolevent/school_event/models/event.py#L216

**Comment by Arpit-SerpentCS** [comment]
@murtuzasaleh Please review the code. Made imnprovements and tested the flow locally according to the comments

cc @JayVora-SerpentCS

**Pull request merged**